dev/tasks/homebrew-formulae/autobrew/apache-arrow.rb

```diff
@@ -19,7 +19,7 @@
 class ApacheArrow < Formula
   desc "Columnar in-memory analytics layer designed to accelerate big data"
   homepage "https://arrow.apache.org/"
-  url "https://www.apache.org/dyn/closer.cgi?path=arrow/arrow-0.15.0.9000/apache-arrow-0.15.0.9000.tar.gz"
+  url "https://www.apache.org/dyn/closer.cgi?path=arrow/arrow-0.15.1.9000/apache-arrow-0.15.1.9000.tar.gz"
   sha256 "9948ddb6d4798b51552d0dca3252dd6e3a7d0f9702714fc6f5a1b59397ce1d28"
```

Comment:
This must be changed also?

Reply:
Yes: the release tests expect that this version match the R package version in `r/DESCRIPTION`. I had to bump that in this PR because, unlike a normal release, the patch release branch wasn't merged, so the R version wasn't bumped.